sdk/firstload/src/index.tsx

```diff
@@ -2,6 +2,7 @@ import {
 	AmplitudeAPI,
 	setupAmplitudeIntegration,
 } from './integrations/amplitude'
+import { SESSION_STORAGE_KEYS } from '@highlight-run/client/src/utils/sessionStorage/sessionStorageKeys'
```

Comment:
i think you need to update https://github.com/highlight/highlight/blob/32faa2a2ad782369c8c11ebeef367e574974a4ed/.reflame.config.jsonc#L245-L252
to fix reflame, cc @lewisl9029

Reply:
Ah yep, I added automatic entry point detection for npm packages, but not for libraries yet. The entry point mapping for libraries is a bit trickier since we have cases like this where entry point names and target locations don't quite match up 1:1.

I've been thinking about how best to address this and I think making libraries act more like workspace packages might be the best way forward in the long term (for instance, making paths relative to the root package directory instead of source (/src) subdirectory), both for compatibility with existing code that makes use of workspaces and to eventually make it possible to auto-configure libraries based on existing workspace configs.

Reply:
Thanks for the tip! Fixing now.